compiler/rustc_middle/src/hir/place.rs

```diff
@@ -81,7 +84,18 @@ pub struct Projection<'tcx> {
 /// A `Place` represents how a value is located in memory.
 ///
 /// This is an HIR version of [`rustc_middle::mir::Place`].
-#[derive(Clone, Debug, PartialEq, Eq, Hash, TyEncodable, TyDecodable, TypeFoldable, HashStable)]
+#[derive(
+    Clone,
+    Debug,
+    PartialEq,
+    Eq,
+    Hash,
+    TyEncodable,
+    TyDecodable,
+    TypeFoldable,
+    TypeVisitable,
+    HashStable
```

Comment:
Can you split these into a separate derive line?

Reply:
Sure, although this file actually had a number of `#[derive]`s that were split over multiple lines prior to this PR; see https://github.com/rust-lang/rust/blob/master/compiler/rustc_middle/src/hir/place.rs#L7-L72.  Should I refactor those at the same time, or in a separate PR?

Reply:
If that's the case, let's just r+ this now

Reply:
Before you said/did that, I had actually already made the change locally.  Since I r-'d due to [the accidental deletion picked up by @SmiteWindows](https://github.com/rust-lang/rust/pull/98206/files#r914383397), the push I've just done has now incorporated those changes/resolved these nits.  There are just a couple more in the codebase that weren't touched by this PR, so I'll open a small refactoring PR to fix those.